servers/cromwell/jobs/test/test_workflow/test_workflow.wdl

```diff
@@ -143,7 +143,7 @@ task TestAdapterSmartSeq2SingleCellTask {
     echo "Int: "${retry_seconds}
     echo "Int: "${timeout_seconds}
 
-    python <<CODE
+    python3 <<CODE
```

Comment:
Impressive comprehensiveness updating Python inside a test WDL's runtime!

FWIW this WDL is unused except as a getting-started example in `servers/cromwell/README.md`, candidate for removal if we want to do more cleanup in a future ticket.